src/libraries/Microsoft.Extensions.Logging.Abstractions/src/LogEntry.cs

```diff
@@ -30,32 +30,32 @@ public LogEntry(LogLevel logLevel, string category, EventId eventId, TState stat
         }
 
         /// <summary>
-        /// Gets the LogLevel
+        /// Gets the LogLevel.
```

Comment:
For consistency with the other properties:

```suggestion
        /// Gets the log level.
```